mizar/dp/mizar/operators/endpoints/endpoints_operator.py

```diff
@@ -324,7 +324,7 @@ def produce_simple_endpoint_interface(self, ep):
             interfaces = InterfaceServiceClient(
                 ep.droplet_obj.main_ip).ProduceInterfaces(InterfacesList(interfaces=interfaces_list))
 
-        logger.info("Produced {}".format(interfaces))
+        logger.info("Endpoint {} Produced {}".format(ep.name, interfaces))
```

Comment:
Can we please add pod_provider to this log in order to get the mapping between pod_name (I assume it holds pod name) and the ep + interface created for the pod?